core/chains/evm/txmgr/eth_resender.go

```diff
@@ -32,6 +30,7 @@ const defaultResenderPollInterval = 5 * time.Second
 // if gas bumping is disabled)
 type EthResender struct {
 	db        *sqlx.DB
```

Comment:
Also remove this variable from here, and from the NewEthResender() params.

Reply:
we need to pass the sqlx.DB in the NewEthResender constructor in order to create the ORM